src/d3d9/d3d9_device.cpp

```diff
@@ -4102,12 +4102,16 @@ namespace dxvk {
 
     const Rc<DxvkBuffer> mappedBuffer = pResource->GetBuffer(Subresource);
 
-    auto formatInfo = imageFormatInfo(pResource->GetFormatMapping().FormatColor);
+    auto& formatMapping = pResource->GetFormatMapping();
+
+    DxvkFormatInfo formatInfo = formatMapping.IsValid()
```

Comment:
Can we keep this a pointer and change UnsupportedThingyBlah?

Reply:
fixed?